dist/localforage-removeitems.es6.js

```diff
@@ -33,30 +33,31 @@ function removeItemsIndexedDB(keys, callback) {
             var dbInfo = localforageInstance._dbInfo;
             var transaction = dbInfo.db.transaction(dbInfo.storeName, 'readwrite');
             var store = transaction.objectStore(dbInfo.storeName);
+            var lastError;
+
+            transaction.oncomplete = function () {
+                resolve();
+            };
+
+            transaction.onabort = transaction.onerror = function () {
+                reject(lastError || transaction.error || 'Unknown error');
+            };
+
+            function requestOnError(evt) {
+                var request = evt.target || this;
+                lastError = request.error || request.transaction.error;
```

Comment:
Something like this?

```js
function requestOnError(evt) {
    var request = evt.target || this;
    if ( !lastError ) lastError = request.error || request.transaction.error;
    reject(lastError);
}
```

I don't really see how it would help, as only first call to `reject` has an effect.